src/main/java/core/basesyntax/exceptions/NotEnoughFruitsInStorageException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exceptions;
+
+public class NotEnoughFruitsInStorageException extends RuntimeException {
```

Comment:
[It's a good practice to create a custom exception if it really needs](https://stackoverflow.com/questions/26984/what-is-the-general-rule-of-thumbs-for-creating-an-exception-in-java) (ex: for specific logic). You can use a simple RuntimeException with an understandable message instead of creating a specific exception that you will use only once in your program.